src/components/SplitPanel/_splitPanel.scss

```diff
@@ -0,0 +1,39 @@
+@import '#scss/vars';
+@import '#scss/utils';
+
+.#{$px}-split-panel {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-medium;
+
+  &:not(.#{$px}-split-panel--borderless) {
+    padding: 2.5rem $spacing-medium;
```

Comment:
I think this needs to be a token for top-bottom padding so it adjusts at breakpoints.

Reply:
Agreed, but The size in figma did not align with any of the tokens they defined and I do not think it changes. I will double check